config/eclipse-java-entando-style.xml

```diff
@@ -1,337 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why are these files being removed?

Reply:
The file was removed from the project because there isn't the need to individually change IDE rules for each project.

Reply:
Those files match the checkstyle included in each project and it can be different from the one in the entando-code-style repository at some point. But I agree that the checkstyle rules are unlikely to change, so I am ok with that.